include/rdma/fabric.h

```diff
@@ -165,6 +165,7 @@ typedef struct fid *fid_t;
 #define FI_COMMIT_COMPLETE	(1ULL << 30)
 #define FI_MATCH_COMPLETE	(1ULL << 31)
 
+#define FI_PEER_TRANSFER	(1ULL << 36)
```

Comment:
Why skip 37?

Reply:
2 reasons.  1. I've had to adjust it 3 times now.  2. Mainly to leave room for PEER_EP.

What I want to do is merge all of the FI_PEER_xxx into just FI_PEER, except for FI_PEER_TRANSFER, which needs special handling.  This use of individual FI_PEER_xxx isn't needed currently.  It's only needed if we report these as cap bits.